fvtest/tril/tril/ilgen.cpp

```diff
@@ -248,6 +248,48 @@ TR::Node* Tril::TRLangBuilder::toTRNode(const ASTNode* const tree) {
          node = TR::Node::create(opcode.getOpCodeValue(), childCount);
          node->setBranchDestination(targetEntry);
      }
+     else if (opcode.isCall()) { 
+        auto compilation = TR::comp();
+
+        const auto addressArg = tree->getArgByName("address");
+        if (addressArg == nullptr) { 
+           TraceIL("  Found call without required address associated\n");
+           return nullptr; 
+        }
+
+        /* I don't want to extend the ASTValue type system to include pointers at this moment,
+         * so for now, we do the reinterpret_cast to pointer type from long
+         */
+        const auto targetAddress = reinterpret_cast<void*>(addressArg->getValue()->get<long>()); 
```

Comment:
Eventually, I would like to see `ASTValue::get()` extended to support pointer types. If not in this PR, please open an issue for it.

Reply:
✅  https://github.com/eclipse/omr/issues/1937